docs/configuring-swc.md

```diff
@@ -192,47 +199,60 @@ Note: This feature requires `v1.2.13+`.
 
 ### jsc.transform.react
 
-- `runtime`
+#### jsc.transform.react.runtime
+
+Possible values: `automatic`, `classic`. This affects how JSX source code will be compiled.
 
-Possible values: `automatic`, `classic`.
+- Use `runtime: automatic` to use a JSX runtime module (e.g. `react/jsx-runtime` introduced in React 17).
+- Use `runtime: classic` to use `React.createElement` instead - with this option, you must ensure that `React` is in scope when using JSX.
 
-Use `automatic` to use new jsx runtimes.
+See https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html
 
-- `importSource`
+#### jsc.transform.react.importSource
 
-Only for `runtime: automatic`. `react` by default.
+When using `runtime: automatic`, determines the runtime library to import.
+
+Defaults to `react`.
 
 This option can be overrided with `@jsxImportSource foo`.
 
-Determines the runtime library to import.
+#### jsc.transform.react.pragma
+
+Replace the function used when compiling JSX expressions.
```

Comment:
Does this only apply to classic runtime?

Reply:
Yes, because `automatic` runtime uses other name for configuration.

Reply:
I will update the doc to clarify this.